external-crates/move/move-core/types/src/lib.rs

```diff
@@ -17,6 +18,7 @@ pub mod parser;
 #[cfg(any(test, feature = "fuzzing"))]
 pub mod proptest_types;
 pub mod resolver;
+pub mod runtime_value;
```

Comment:
It looks like the normal `value` module has been almost entirely gutted? Maybe we should delete it and move the functions elsewhere?

Reply:
It has! I only kept is as a file to hold common things between the two, but happy to looking in to removing it :)

Reply:
`value.rs` is no more!